### PR TITLE
More timeout HPC-5325

### PR DESCRIPTION
**Pull request by JensTimmerman**
added more options for timing out, they work (at least in some cases)

### CODE REVIEW

File: lib/vsc/manage/managecommands.py [hunk 1]
```diff
@@ -239,8 +240,17 @@ def exec_command(self, command, bufsize=-1, timeout=get_config("COMMAND_TIMEOUT"
             stdin = chan.makefile('wb', bufsize)
             stdout = chan.makefile('rb', bufsize)
             stderr = chan.makefile_stderr('rb', bufsize)
-            exitcode = chan.recv_exit_status()
-            return stdin, stdout, stderr, exitcode
+            start = datetime.datetime.now()
+            while datetime.datetime.now() - start < datetime.timedelta(0, timeout):
```
Comment (itkovian) [inline lib/vsc/manage/managecommands.py]: Can you use named arguments for the time delta call, since it's so frakked up?

**Comment by JensTimmerman** [comment]
please rereview @itkovian

File: lib/vsc/manage/managecommands.py [hunk 2]
```diff
@@ -277,6 +287,7 @@ def run(self):
         # run the command (with a timeout)
         try:
             stdin, stdout, stderr, exitcode = ssh.exec_command(self.command, timeout=self.timeout)
+            self.log.debug('ran ssh.exec_command')
```
Comment (itkovian) [inline lib/vsc/manage/managecommands.py]: Add the timeout value passed?

**Comment by itkovian** [comment]
If you think that extra bit of info is not needed,I'm OK with merging.

File: lib/vsc/manage/managecommands.py (hunk 2 above)
Comment (JensTimmerman) [inline lib/vsc/manage/managecommands.py]: this is already logged before running the command, one line higher, going to run....

**Pull request merged**